capa/features/extractors/binja/find_binja_api.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2020 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import subprocess
+
+# When the script gets executed as a standalone executable (via PyInstaller), `import binaryninja` does not work because
+# we have excluded the binaryninja module in `pyinstaller.spec`. The trick here is to call the system Python and try
```

Comment:
> system python

i wonder if this works with virtualenvs. at first i thought "no", but i think if the virtualenv is activated then "python.exe" would refer to the virtualenv python and things would work as you'd want. 

in any case, this is a clever solution. i'm a little hesitant that there may be edge cases and weird things to debug from users, but let's evaluate when we get that feedback.

Reply:
great, thanks for confirming!